### ISSUE TITLE
Nuklear Unity Loader

### ISSUE THREAD

**Issue by oistikbal** [Immediate-Mode-UI/Nuklear#536]
Hi, İ imported Nuklear for Unity.  There are currently 4 repos. The main loader is a library with an API to create another library to load in Unity. The other 2 package repos are for the interaction with the unity.

it's currently hardcoded to directx 11 but it will be updated in time.

https://github.com/Chutpot

**Comment by oistikbal** [comment]
I'm confused right now what to support, i realized that i could easily use NuklearDotNet wrapper directly in my project, it is still like a c api but it can be used.

The upper ui get called in c++ and the other at one c#.
![4123214](https://user-images.githubusercontent.com/12850964/212079400-7348be8a-4268-4d20-bee4-f27db4aff654.PNG)

**Comment by dumblob** [comment]
Yep, that is the flexibility a pure C architecture offers :wink:.

**Comment by RobLoach** [comment]
Maybe link this in the Integrations page in the wiki?
https://github.com/Immediate-Mode-UI/Nuklear/wiki/Integrations
